service/src/test/java/bio/terra/workspace/app/configuration/external/AzureStateEnabledTest.java

```diff
@@ -13,6 +14,7 @@
   @Autowired private AzureState azureState;
 
   @Test
+  @Tag("azure")
```

Comment:
I intended this to be a unit test, not an azure test, to make sure that the explicit profile cases got the right answer.
Perhaps remove the `@Tag` and add a comment that even though this has azure written all over it, it is supposed to be a unit test.